cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -217,31 +219,35 @@ def test_v2v_ui_set2(appliance, v2v_providers, form_data_single_datastore, soft_
     view.options.run_migration.select("Save migration plan to run later")
     # Test Create migration plan -Create and Read
     view.options.create.click()
+    view.results.close.click()
     # Test plan has unique name
-    view = navigate_to(migration_plan_collection, 'Add')
+    view = appliance.browser.create_view(MigrationDashboardView)
+    view.wait_displayed()
+    view.create_migration_plan.click()
+    view = appliance.browser.create_view(AddMigrationPlanView)
+    view.general.wait_displayed()
     view.general.infra_map.select_by_visible_text(mapping.name)
     view.general.name.fill(plan_name)
     view.general.description.fill(fauxfactory.gen_string("alphanumeric", length=150))
     # following assertion will fail in 5.9.4 as they have not backported this change to 5.9.4
     soft_assert(view.general.name_help_text.read() == 'Please enter a unique name')
+    view.cancel_btn.click()
 
-    view = navigate_to(migration_plan_collection, 'All')
+    view = appliance.browser.create_view(MigrationDashboardView)
     soft_assert(plan_name in view.migration_plans_not_started_list.read())
-    soft_assert(view.infra_mapping_list.get_associated_plans(mapping.name) == plan_name)
     soft_assert(view.migration_plans_not_started_list.get_plan_description(plan_name) ==
      plan_description)
     # Test Associated Plans count  correctly Displayed in Map List view
     soft_assert(str(len(vm_selected)) in view.migration_plans_not_started_list.
         get_vm_count_in_plan(plan_name))
+    view.browser.refresh()
```

Comment:
Question : Why is this required ?

Reply:
This is required because the Associated plans do not show up in infra-mappings-list before refresh.